Libs/MRML/Widgets/qMRMLNodeComboBox.cxx

```diff
@@ -43,6 +43,15 @@
 #include <vtkMRMLNode.h>
 #include <vtkMRMLScene.h>
 
+// ----------------------------------------------------------------------------
+
+namespace
+{
+  const QString createNew = QObject::tr("Create new ");
+  const QString asSuffix = QObject::tr(" as...");
+  const QString renameTitlePrefix = QObject::tr("Rename ");
```

Comment:
I don't know enough about all the languages possibly involved here, but it seems to me some languages may not structure the sentence the same way and these may not be easy to break down as prefix and suffix.